lib/init.js

```diff
@@ -0,0 +1,141 @@
+'use babel';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/extensions
+import { CompositeDisposable } from 'atom';
+
+let path;
+let helpers;
+
+const pattern = '.+:(?<line>\\d+):(?<col>\\d+)-(?<colEnd>\\d+):' +
+  ' \\((?<type>[EW])\\d+\\) (?<message>.*)';
+
+function loadDeps() {
+  if (!helpers) {
+    helpers = require('atom-linter');
+  }
+  if (!path) {
+    path = require('path');
+  }
+}
+
+function checkedOption(params, opt, args) {
+  if (args.length > 0) {
+    params.push(opt);
+    return params.concat(args);
+  }
+
+  return params;
+}
+
+function reportToMessage(report, file) {
+  const end = report.range[1];
```

Comment:
So really this is actually a question. Looking back at the regex being sent to `parse`, it looks like `luacheck` actually gives an _end_ column as well... I assumed that it wasn't due to the line below this that adds 1 to it. If `luacheck` is actually giving us an end column that should just be used directly.

Does `luacheck` _always_ give the column range?